docs/source/development.rst

```diff
@@ -109,6 +109,9 @@ In the first terminal, let's create our database and then run the python webserv
     cd ~/dev/hawc
     source ./venv/bin/activate
 
+    # install latest packages and requirements from code
+    make sync-dev
+
     # sync db state with application state
     manage.py migrate
```

Comment:
Technically we can remove this step from Linux/Windows instructions now, but theres no harm in keeping it either.